src/main/java/com/scalar/cassy/CassyClient.java

```diff
@@ -0,0 +1,115 @@
+package com.scalar.cassy;
+
+import com.scalar.cassy.rpc.BackupListingRequest;
+import com.scalar.cassy.rpc.BackupListingResponse;
+import com.scalar.cassy.rpc.BackupRequest;
+import com.scalar.cassy.rpc.BackupResponse;
+import com.scalar.cassy.rpc.CassyGrpc;
+import io.grpc.Channel;
+import io.grpc.ManagedChannelBuilder;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+
+public class CassyClient {
+  private final CassyGrpc.CassyBlockingStub blockingStub;
+
+  public CassyClient() {
+    this(ManagedChannelBuilder.forAddress("localhost", 20051).usePlaintext());
```

Comment:
Let's make the cassy server host and port configurable with the help of an environment variable.